p2p/host/autorelay/autorelay_test.go

```diff
@@ -440,13 +443,14 @@ func TestMinInterval(t *testing.T) {
 		autorelay.WithMinCandidates(2),
 		autorelay.WithNumRelays(1),
 		autorelay.WithBootDelay(time.Hour),
-		autorelay.WithMinInterval(500*time.Millisecond),
+		autorelay.WithMinInterval(2*time.Second),
 	)
 	defer h.Close()
 
-	cl.Add(500 * time.Millisecond)
```

Comment:
Why these changes?

Reply:
so previously the initial call to peer source was triggered without waiting for any time to progress on the clock by the scheduledWork function which runs right before we go into the for loop in background go routine. 

With this change with a mock clock we need to increment the clock once after the background go routine has initialised the timer for peerSourceCallback.